specification/signalr/data-plane/AzureSignalR/v1/azuresignalr.json

```diff
@@ -0,0 +1,710 @@
+{
+    "openapi": "3.0.1",
+    "info": {
+      "title": "Azure SignalR Service REST API",
+      "version": "v1"
+    },
+    "paths": {
+      "/api/v1/health": {
+        "get": {
+          "description": "Get service health status",
+          "tags": [],
+          "responses": {
+            "200": {
+              "description": "Ok"
+            },
+            "503": {
+              "description": "ServiceUnavailable"
```

Comment:
We suggest to not document error, or to mark them with `x-ms-error-response` so the codegen knows this is an error. Otherwise SDK here will consider a 503 a success, which is unlikely what you want.
https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-error-response

Reply:
Nice catch, I will remove all the 5xx error codes.